plugins/workspace-backpack/src/backpack.js

```diff
@@ -817,3 +821,59 @@ Blockly.Css.register(`
   opacity: 0.8;
 }
 `);
+
+
+/**
+ * Custom serializer so that the backpack can save and later recall which
+ * blocks have been saved in a workspace.
+ */
+class BackpackSerializer {
+  /** Constructs the backpack serializer */
+  constructor() {
+    /**
+     * The priority for deserializing block suggestion data.
+     * Should be after blocks, procedures, and variables.
+     * @type {number}
+     */
+    this.priority = Blockly.serialization.priorities.VARIABLES - 10;
```

Comment:
If you want this to be after everything it should be `Blockly.serialization.priorities.BLOCKS - 10`, since blocks are last =)

You can see all of the built-in priority values here: https://github.com/google/blockly/blob/develop/core/serialization/priorities.ts